src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder getSalaryOutput = new StringBuilder();
+        int[] salary = new int[names.length];
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Since the formatter is used multiple times in the code, it should be made into a constant field. Define it as a private static final field at the class level and use an uppercase name to follow Java naming conventions for constants.